modules/apps/portlet-configuration/portlet-configuration-css-web/src/main/resources/META-INF/resources/js/index.js

```diff
@@ -8,7 +8,7 @@ import React, {useState} from 'react';
 
 const HEX_COLOR_REGEX = /^#?[0-9A-F]{3}(?:[0-9A-F]{3})?$/i;
 
-export function ColorPicker({color, label, name}) {
+export function ColorPickerInput({color, label, name}) {
```

Comment:
I have sent a fix for this: https://github.com/liferay-echo/liferay-portal/pull/15147 should I close the PR? or we can remove this commit?

Reply:
@victorg1991 feel free to keep your PR open as it'll likely get merged first. I will resolve conflicts before sending to brian